src/components/RefineSearch/RefineSearch.tsx

```diff
@@ -36,32 +45,22 @@ const RefineSearch = ({
   setAppliedFilters,
 }: RefineSearchProps) => {
   const router = useRouter()
-
-  const fields = [
-    { value: "materialType", label: "Format" },
-    { value: "language", label: "Language" },
-    { value: "dateAfter", label: "Start Year" },
-    { value: "dateBefore", label: "End Year" },
-    { value: "subjectLiteral", label: "Subject" },
-  ]
-
-  const dateFieldset = (
-    <FieldsetDate
-      onDateChange={(dateField: DateFormName, data: string) => {
-        // update the parent state to know about the updated dateValues
-        setAppliedFilters((prevFilters) => {
-          return {
-            ...prevFilters,
-            [dateField]: [data],
-          }
-        })
-      }}
-      appliedFilters={{
-        dateBefore: appliedFilters.dateBefore && appliedFilters.dateBefore[0],
-        dateAfter: appliedFilters.dateAfter && appliedFilters.dateAfter[0],
-      }}
-    />
-  )
+  const dateInputRef = useRef<TextInputRefType>()
+  const { dateFormProps, validateDateRange } = useDateForm({
+    changeHandler: (e: SyntheticEvent) => {
+      const target = e.target as HTMLInputElement
+      // update the parent state to know about the updated dateValues
+      setAppliedFilters((prevFilters) => {
+        return {
+          ...prevFilters,
+          [target.name]: [target.value],
+        }
+      })
+    },
+    inputRefs: [dateInputRef],
```

Comment:
Why is only one set?

Reply:
Hmm I was thinking there was different behavior from RefineSearch to Advanced search, but I'm seeing now not only is it the same, but the focus does automatically go to the first field, even when there is only one ref. Maybe I will switch it back to only using one ref across both components.

Reply:
I'm seeing some unexpected behavior around this. I'd love to run it by you on monday.